File: SCDemo/LangImageAnimLib/src/main/java/net/lang/animation/ImageFrame.java
package net.lang.animation;

import android.graphics.Bitmap;

public class ImageFrame {
    private Bitmap mImage;
    private int mDelay;
    private ImageFrame mNextFrame;

    public ImageFrame(Bitmap image, int delay) {
        this.mImage = image;
        this.mDelay = delay;
        this.mNextFrame = null;
    }

    public Bitmap getImage() {
        return this.mImage;
    }

    public int getDelay() {
        return this.mDelay;
    }

    public void setNextFrame(ImageFrame imageFrame) {
        this.mNextFrame = imageFrame;
    }

    public ImageFrame getNextFrame() {
        return this.mNextFrame;
    }
}

File: SCDemo/LangStreamer1Kit/src/main/java/net/lang/streamer/faceu/image/ImageFrame.kt
package net.lang.streamer.faceu.image

import android.graphics.Bitmap

/**
 * Created by Rayan on 2019/7/17.
 */

class ImageFrame(private var im: Bitmap, private var del: Int) {

    var image: Bitmap? = im

    var delay = del

    var nextFrame: ImageFrame? = null
}